src/test/components/__snapshots__/MarkerTable.test.js.snap

```diff
@@ -213,7 +213,6 @@ exports[`MarkerTable renders some basic markers and updates when needed 1`] = `
               class="treeViewBodyInner treeViewBodyInner1InnerChunk"
             >
               <div
-                aria-label="foobar, running time is undefined (undefined), self time is undefined"
```

Comment:
Can you please file a follow-up to implement aria labels on the marker table, if needed?